smartsim/_core/control/controller.py

```diff
@@ -48,12 +46,7 @@
     unpack_colo_fs_identifier,
     unpack_fs_identifier,
 )
-from ..._core.utils.redis import (
-    fs_is_active,
-    set_ml_model,
-    set_script,
-    shutdown_fs_node,
-)
```

Comment:
I know this doesn't `really` matter since we're probably going to remove or seriously refactor this module shortly, but just be warned that these symbols are no longer defined, but still used in this module and our CI is not going to like that.

Consider making a couple of mocks or removing these symbols usage

Reply:
make mocks